src/views/network/AntennaCriteriaCheck.vue

```diff
@@ -426,7 +426,7 @@ export default {
           this.netcommies = netcomMembersResponse.data.data.map(netcommie => ({
             user_id: netcommie.user_id,
             first_name: netcommie.user.first_name,
-            email: netcommie.user.email
+            email: netcommie.user.gsuite_id ? netcommie.user.gsuite_id : netcommie.user.email
```

Comment:
```suggestion
            email: netcommie.user.notification_email
```
This would be in line with other functionality in MyAEGEE. Would that also be okay?

Reply:
As discussed internally, we'll go with this change as GSuite email is preferred even with a changed notification email